src/main/java/spireCafe/interactables/merchants/fleamerchant/FleaMarketRelicArticle.java

```diff
@@ -0,0 +1,125 @@
+import com.megacrit.cardcrawl.relics.*;
+import com.megacrit.cardcrawl.shop.ShopScreen;
+import com.megacrit.cardcrawl.vfx.RelicAboveCreatureEffect;
+import spireCafe.Anniv7Mod;
+import spireCafe.abstracts.AbstractArticle;
+import spireCafe.abstracts.AbstractMerchant;
+import spireCafe.interactables.merchants.secretshop.SecretShopMerchant;
+import spireCafe.util.TexLoader;
+
+import java.util.ArrayList;
+
+public class FleaMarketRelicArticle extends AbstractArticle {
+    private static final String ID = Anniv7Mod.makeID(FleaMarketRelicArticle.class.getSimpleName());
+    private static final UIStrings uiStrings = CardCrawlGame.languagePack.getUIString(ID);
+    private static final float RELIC_Y = 364.0F * Settings.scale;
+    private final AbstractRelic relic;
+    private boolean primed;
+    private HaggleArticle haggleArticle;
+    private final double priceJitter;
+    private int price;
+
+    public FleaMarketRelicArticle(AbstractMerchant merchant, int slot, HaggleArticle haggleArticle) {
+        super(ID, merchant, 964.0F * Settings.xScale + 150.0F * slot * Settings.xScale, RELIC_Y, ImageMaster.loadImage("images/relics/"+new PenNib().imgUrl));
+        this.primed = slot==2;
+        this.haggleArticle = haggleArticle;
+        this.relic = primed?getRandomPrimedRelic():getRandomUsedRelic();
+        this.itemTexture = new TextureRegion(ImageMaster.loadImage("images/relics/"+relic.imgUrl));
+        priceJitter = AbstractDungeon.merchantRng.random(0.95F, 1.05F);
+    }
+
+    @Override
+    public void onClick() {
+        if (!canBuy()) {
+            ((FleaMerchant) merchant).cantBuy();
+        } else {
+            ((FleaMerchant) merchant).sold();
+        }
+        super.onClick();
+    }
+
+    @Override
+    public boolean canBuy() {
+        return AbstractDungeon.player.gold > getModifiedPrice();
+    }
+
+    @Override
+    public void onBuy() {
+        AbstractDungeon.player.loseGold(getModifiedPrice());
+        AbstractDungeon.getCurrRoom().relics.add(relic);
+        relic.instantObtain(AbstractDungeon.player, AbstractDungeon.player.relics.size(), false);
+        relic.flash();
+    }
+
+    @Override
+    public String getTipHeader() {
+        return uiStrings.TEXT[0] + relic.name;
+    }
+
+    @Override
+    public String getTipBody() {
+        return uiStrings.TEXT[primed?1:3] + relic.counter + uiStrings.TEXT[primed?2:(relic.counter==1?4:5)] + relic.description;
+    }
+
+    @Override
+    public int getBasePrice() {
+        return price;
+    }
+
+    public AbstractRelic getRandomPrimedRelic(){
+        ArrayList<AbstractRelic> primedRelics = new ArrayList<>();
+        ArrayList<Integer> prCounters = new ArrayList<>();
+        primedRelics.add(new PenNib());         prCounters.add(10);
+        primedRelics.add(new Nunchaku());       prCounters.add(10);
+        primedRelics.add(new InkBottle());      prCounters.add(10);
+        primedRelics.add(new IncenseBurner());  prCounters.add(6);
```

Comment:
Having a 1 in 6 chance to get Incense Burner is pretty nice! Though if there are many different merchants, Flo only showing up 1 out of every N times you visit the cafe means this probably isn't an issue.